src/main/scripts/org/reaktivity/specification/socks/client.connect.send.data/client.rpt

```diff
@@ -29,13 +29,11 @@ write [0x05 0x01 0x00]
 read [0x05 0x00]
 
 ## Write CONNECT request
```

Comment:
I don't see 0xFE in RFC spec:  https://www.ietf.org/rfc/rfc1928.txt.  I suspect you want domain name or ip4 or ipv6 (Note, domain name does not include the scheme).  I would also suggest clarifying this test commenting on what each field means, maybe:

```
write [0x05]     # version: SOCKS v5
write [0x01]     # CONNECT command
write [0x00]   # RSV field
write [0x03]   # Address Type domain name
...

```

Reply:
or 

```
write [0x05]     # version: SOCKS v5
      [0x01]     # CONNECT command 
      [0x00]   # RSV field
      [0x03]   # Address Type domain name
```